### PR TITLE
Follow natspec for documentation in TeleporterMessenger

### PR DESCRIPTION
**Pull request by geoff-vball**
## Why this should be merged
Closes https://github.com/ava-labs/teleporter/issues/299

## How this works
Follows NatSpec format detailed [here](https://docs.soliditylang.org/en/latest/natspec-format.html)

We don't actually currently do anything with the documentation generated in `TeleporterMessenger.json`.

## How this was tested
N/A

## How is this documented
N/A

### CODE REVIEW

File: contracts/src/Teleporter/TeleporterMessenger.sol [hunk 1]
```diff
@@ -36,54 +36,56 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     using SafeERC20 for IERC20;
     using ReceiptQueue for ReceiptQueue.TeleporterMessageReceiptQueue;
 
-    // SentMessageInfo includes the fee information for a given message submitted
-    // to be sent, along with the hash of the message itself.
+    /// @notice SentMessageInfo includes the fee information for a given message submitted
```
Comment (michaelkaplan13) [inline contracts/src/Teleporter/TeleporterMessenger.sol:39]: Could you update these to be:
```
/**
 * @notice ...
 */
```
to match the other multi-line natspec comments here?

File: contracts/src/Teleporter/TeleporterMessenger.sol [hunk 2]
```diff
@@ -36,54 +36,76 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     using SafeERC20 for IERC20;
     using ReceiptQueue for ReceiptQueue.TeleporterMessageReceiptQueue;
 
-    // SentMessageInfo includes the fee information for a given message submitted
-    // to be sent, along with the hash of the message itself.
+    /**
+     *  @notice SentMessageInfo includes the fee information for a given message submitted
```
Comment (michaelkaplan13) [inline contracts/src/Teleporter/TeleporterMessenger.sol:40]: Super nit: There's an extra space on each line in the new comments compared to the other natspec comments in this contract.

Comment (geoff-vball) [reply contracts/src/Teleporter/TeleporterMessenger.sol:40]: Oh yeah. No problem!

**Review by michaelkaplan13 — APPROVED** [review]
🙏

**Review by minghinmatthewlam — APPROVED** [review]
LGTM

**Pull request merged**